### ISSUE TITLE
Move maps to their own project repos

### ISSUE THREAD

**Issue by maptastik** [LexHousingStudies/LexHousingStudies.github.io#4]
## What

We should move each map to its own individual project repo a la [grapi-2013](https://github.com/LexHousingStudies/grapi-2013) or the [daniels-sales](https://github.com/LexHousingStudies/daniels-sales).
## Why

Currently several of our maps exist as directories in this repo. That's fine, but there are two slightly different types of content within this repo now.
1. Our website
2. Thematic maps

Being in the same repo means that a change to either a map or a the website means syncing the entire repo. There's just a lot going on in this repo that could potentially result in conflicts. I think it would be better if each map were it's own individual project repo and this repo would be solely for the website. This would have the added benefit of making the code for individual maps easier to share. For instance, if someone wants to fork our abandonment map, they are going to get all the other maps and our website code.
## How to do this
1. Create individual repos for each map with the exact same name is its directory in this repo.
2. Within those individual repos, create a `gh-pages` branch.
3. Make the `gh-pages` branch the default branch
4. Clone each individual project repo
5. Copy the files for each of the maps in this repo to their project repo
6. Remove the maps from this repo
7. Push changes

**Comment by maptastik** [comment]
@lfgreenfield I think this would affect you the most of anyone else. Any thoughts or questions about this?

**Comment by lfgreenfield** [comment]
Ryan, I think this is a great idea. It makes sense. I just don't follow the
steps to do this but would be interested in learning from you. Any chance
we'd be able to sit down together and you could walk me through this
process? Always iffy when it comes to using different branches.

Laura F. Greenfield
lfgr225@g.uky.edu
(859) 361-8474

_University of Kentucky_
Lexington, Kentucky
Pursuing B.A. in Geography
_Oakland Farm Trees_
Paris, Kentucky
www.oaklandfarmtrees.com

**Comment by maptastik** [comment]
@lfgreenfield Yeah, sorry if the steps aren't super clear. I was falling asleep while writing them. But let's meetup and walk through this, for sure.
